### ISSUE TITLE
Session drop-down missing in docked debug action bar

### ISSUE THREAD

**Issue by weinand** [Microsoft/vscode#50787]
testing 49892:

- debug a multi session scenario:

Observe:
the session drop down is missing if the debug action bar is "docked":

![2018-05-30_15-15-48](https://user-images.githubusercontent.com/1898161/40722499-ce0e72a0-641c-11e8-99cb-27cc388adc4d.png)

This happens even if there is enough space for it.

**Comment by isidorn** [comment]
Yes this was a design decision during a standup. The reason for that being that when the debug toolbar is docked the call stack can be seen and all the information is there.
The reason for not showing the drop down is due to the limitied horizontal space.

I will treat this as a bug to always show clear indication in the CALL STACK what is the focused session

**Comment by weinand** [comment]
But you are not showing the session drop down even if there is enough space...

Another reason why I don't like the missing session drop down is that the CALL STACK view is too far away from the action bar. All actions work on the focussed session but what that session is, is "far away" from the action buttons.

In light of this maybe the DEBUG SESSIONS view (proposed by @bpasero) makes more sense to me: the section header would show the action buttons and the section's content would show the focussed session...

**Comment by bpasero** [comment]
I thought we would still show the session dropdown if you make the view large enough until we have decided on showing a session view or not. I remember seeing such a design in one of the standups @isidorn

**Comment by isidorn** [comment]
The original design was showing the session dropdown always but it would be the first to shrink when space is limited. The feedback for this was that is looks too ugly and feels like duplication from the call stack view.
I can bring back this design for this milestone. Let me know what you think
Otherwise we can just go with the current approach and wait for user feedback.

**Comment by bpasero** [comment]
@isidorn I have seen it, maybe record a little screencast so that @weinand can also see it in action

**Comment by isidorn** [comment]
![debug](https://user-images.githubusercontent.com/1926584/40729057-d2400df2-642b-11e8-8f8b-72dd75312b9d.gif)

**Comment by weinand** [comment]
The resize behaviour looks fine to me.
And I don't consider it "duplication from the call stack view". 
Instead I would call the current behaviour a "loss of functionality". 

I'm fine with shipping whatever we have,
but I would call both versions an "early experiments"...

**Comment by isidorn** [comment]
Since the resizing behavior is not that bad and since I feel like you both lean towards showing it I have decided to go with that solution.
Thus closing this issue nad let's wait for user feeback and see.
Thanks for feedback!

**Comment by weinand** [comment]
So let's call this feature "Preview" then.
